modules/sharedaddy/recaptcha.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/**
+ * Class that handles reCAPTCHA.
+ */
+class Jetpack_ReCaptcha {
+
+	/**
+	 * URL to which requests are POSTed.
+	 *
+	 * @const string
+	 */
+	const VERIFY_URL = 'https://www.google.com/recaptcha/api/siteverify';
+
+	/**
+	 * Site key to use in HTML code.
+	 *
+	 * @var string
+	 */
+	private $site_key;
+
+	/**
+	 * Shared secret for the site.
+	 *
+	 * @var string
+	 */
+	private $secret_key;
+
+	/**
+	 * Config for reCAPTCHA instance.
+	 *
+	 * @var array
+	 */
+	private $config;
+
+	/**
+	 * Error codes returned from reCAPTCHA API.
+	 *
+	 * @see https://developers.google.com/recaptcha/docs/verify
+	 *
+	 * @var array
+	 */
+	private $error_codes = array(
```

Comment:
Should we translate these and any other strings in this class? Even though we're not using them directly now, another module could use them in the future.